smarts/env/hiway_env.py

```diff
@@ -142,34 +145,22 @@ def __init__(
         if visdom:
             visdom_client = VisdomClient()
 
-        all_sumo = Scenario.supports_traffic_simulation(scenarios)
-        traffic_sim = None
-        if not all_sumo:
-            # We currently only support the Native SUMO Traffic Provider and Social Agents for SUMO maps
-            if zoo_addrs:
-                warnings.warn("`zoo_addrs` can only be used with SUMO scenarios")
-                zoo_addrs = None
-            warnings.warn(
-                "We currently only support the Native SUMO Traffic Provider and Social Agents for SUMO maps."
-                "All scenarios passed need to be of SUMO, to enable SUMO Traffic Simulation and Social Agents."
-            )
-            pass
-        else:
-            from smarts.core.sumo_traffic_simulation import SumoTrafficSimulation
-
-            traffic_sim = SumoTrafficSimulation(
+        smarts_traffic = LocalTrafficProvider(endless_traffic=endless_traffic)
+        traffic_sims = [smarts_traffic]
+        if Scenario.all_support_sumo_traffic(scenarios):
```

Comment:
Good point.  I decided to just go ahead and start the Sumo traffic simulator whenever there's a scenario with a Sumo map, just in case it might be used.  I think it shouldn't hurt to have it around in that case.